tests/postgres_tests/test_indexes.py

```diff
@@ -1,8 +1,33 @@
-from django.contrib.postgres.indexes import GinIndex
+from django.contrib.postgres.indexes import BRINIndex, GinIndex
 from django.db import connection
+from django.test import skipUnlessDBFeature
 
 from . import PostgreSQLTestCase
-from .models import IntegerArrayModel
+from .models import CharFieldModel, IntegerArrayModel
+
+
+class BRINIndexTests(PostgreSQLTestCase):
+
+    @skipUnlessDBFeature('has_brin_index_support')
+    def test_repr(self):
+        index = BRINIndex(fields=['title'])
+        self.assertEqual(repr(index), "<BRINIndex: fields='title'>")
+
+    @skipUnlessDBFeature('has_brin_index_support')
+    def test_eq(self):
```

Comment:
Yes. I think this requires some addition to the introspection of the indexes? I've experimented with this. The key `expression` is `None` for the index.